cli/args.go

```diff
@@ -39,6 +39,9 @@ func ParseTerragruntOptions(cliContext *cli.Context) (*options.TerragruntOptions
 // and look for the ones we need, but in the future, we should change to a different CLI library to avoid this
 // limitation.
 func parseTerragruntOptionsFromArgs(terragruntVersion string, args []string, writer, errWriter io.Writer) (*options.TerragruntOptions, error) {
+
+	// We need to set proper writer early on
+	util.GlobalFallbackLogEntry.Logger.SetOutput(errWriter)
```

Comment:
I think we want the logrus stuff to default to stderr as soon as you create it. That way, even if there is an error in `main.go`, before we get here, it still logs to `stderr`.

Reply:
Okay, will push a change shortly

Reply:
Thx!